### ISSUE TITLE
Please allow using a system variable for the garage doorstate

### ISSUE THREAD

**Issue by bgewehr** [thkl/homebridge-homematic#668]
**Is your feature request related to a problem? Please describe.**
My Garage does use a different way to sense the doorstate. I was able to put the result in a system variable in my CCU. 

**Describe the solution you'd like**
I would llike to see the garage accessory being able to not only request states of CCU devices but also CCU system variables to detect the doorstate.

**Describe alternatives you've considered**
I tried every method I could think of and was not able to achieve a working setup with the current possibilities.

**Comment by Tonitom** [comment]
Yes, that would be great, I have also tried with CUxD, but there the status is not updated.

Tonitom
